dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05225RemoveLoadRecordsToIndex.java

```diff
@@ -0,0 +1,75 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import java.util.List;
+
+/**
+ * Remove {@code load_records_to_index} stored procedure
```

Comment:
I think we should add here detail about what this task do

for example

- remove the load_records_to_index function if it exists